wfe2/wfe.go

```diff
@@ -825,6 +851,18 @@ func (wfe *WebFrontEndImpl) NewAccount(
 		return
 	}
 	newRegistrationSuccessful = true
+
+	emails := contactsToEmails(accountCreateRequest.Contact)
+	if wfe.ee != nil && len(emails) > 0 {
+		_, err := wfe.ee.SendContacts(ctx, &emailpb.SendContactsRequest{
+			// Note: We are explicitly using the contacts provided by the
+			// subscriber here, rather than the contacts returned by the RA.
+			Emails: emails,
+		})
+		if err != nil {
+			wfe.log.Warningf("Error creating contact: %v", err)
+		}
```

Comment:
I think this should return an error instead of just logging it. It's true we want to insulate our availability from the availability of the Pardot API, but that's achieved by the queuing inside the email exporter. If our internal setup is broken (i.e. the email exporter is down), we should surface that in the plainest way possible.